cli/src/lib.rs

```diff
@@ -143,7 +143,7 @@ fn run_until_exit<T, C, W>(
 	let _telemetry = service.telemetry();
 
 	let work = worker.work(&*service, Arc::new(executor));
-	let _ = runtime.block_on(service.select(work));
+	let _ = runtime.block_on(service.map_err(|_| ()).select(work));
```

Comment:
Is service now returning a different error type than `()`?

Reply:
Yes, a `ServiceError` apparently: https://github.com/paritytech/substrate/pull/3353/files#diff-d2b444bb5d594e411359291f8be4fe56

Reply:
Why do we not want to at least log this error?